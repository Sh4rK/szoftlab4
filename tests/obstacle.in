loadMap test.map
loadMission attack.mission
buildObstacle 10 10
listTowers
step 15
listEnemies
listProjectiles
step 5
listEnemies
listProjectiles
step 5
listEnemies
listProjectiles
step 5
listEnemies
listProjectiles
step 5
listEnemies
listProjectiles
step 5
listEnemies
listProjectiles
step 5
listEnemies
listProjectiles
step 5
listEnemies
listProjectiles

exit
