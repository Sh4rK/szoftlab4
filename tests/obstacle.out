1 75.0 (3.6;3.6)
1 75.0 (4.9;4.9)
1 75.0 (6.2;6.2)
1 75.0 (7.0;7.0)
1 75.0 (7.6;7.6)
1 75.0 (8.3;8.3)
1 75.0 (8.9;8.9)
1 75.0 (9.6;9.6)
